deploy/provider-gcp-plugin.yaml

```diff
@@ -68,6 +68,11 @@ spec:
         app: csi-secrets-store-provider-gcp
     spec:
       serviceAccountName: secrets-store-csi-driver-provider-gcp
+      tolerations:
+        - key: kubernetes.io/arch
+          operator: Equal
+          value: arm64
+          effect: NoSchedule
```

Comment:
do you mind reverting this from the CL for now. Building an arm64 version LGTM but likely will want this as some parameter into a helm-chart install since thats how it looks like https://github.com/kubernetes-sigs/secrets-store-csi-driver does it